agent/src/beerocks/slave/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ file(GLOB beerocks_agent_sources
     # [TASK] Move link metric related classes to BPL #910
     ${MODULE_PATH}/link_metrics/*.c*
     ${MODULE_PATH}/platform_manager/*.c*
+    ${MODULE_PATH}/tasks/*.c*
```

Comment:
I really prefer that we switch for explicit naming of the files we want to be part of the build.
But OK for now to keep the same method and change it all together later.

Reply:
I have a lot against GLOB, but my opinion is not important.
The developers of CMake [recommend](https://cmake.org/cmake/help/latest/command/file.html#filesystem) not to use GLOB.

>  Note: We do not recommend using GLOB to collect a list of source files from your source tree.

They give one reason that might be irrelevant to us, but there are more reasons.